release-manager/ocp-config/cd-docgen.yml

```diff
@@ -3,6 +3,25 @@ kind: Template
 labels:
   template: cd-docgen
   app: '${PROJECT}-docgen'
+parameters:
+  - name: PROJECT
+    displayName: Application
+    description: The name of the application project.
+    required: true
+  - name: BITBUCKET_URL
+    displayName: Application
+    description: The URL of your bitbucket server.
+    required: true
+  - name: MEMORY_LIMIT
+    displayName: Memory Limit
+    description: Maximum amount of memory available for the container.
+    value: 512Mi
+    required: true
+  - name: MEMORY_REQUEST
```

Comment:
we could consider adding CPU limits also

Reply:
Don't be overly aggressive on both memory and CPU unless you understand the usage patterns, please. Have you tried out these values with some actual documents being generated?

Reply:
shall then be increased to 1Gi? do you know if it has gone over 1Gi usage when using it?

Reply:
We haven't measured this in detail yet, as the Groovy based implementation will be replaced with a Golang one in v3 anyway. Anyway, we should avoid anything that impacts service availability until we have that (see below). I'll look up the current settings on the OCTEST cluster.